apple/RNSVGRenderable.mm

```diff
@@ -250,7 +250,10 @@ - (void)renderTo:(CGContextRef)context rect:(CGRect)rect
 
   [self beginTransparencyLayer:context];
 
-  if (self.mask || self.filter) {
+  RNSVGFilter *filterNode = self.filter ? (RNSVGFilter *)[self.svgView getDefinedFilter:self.filter] : nil;
+  RNSVGMask *maskNode = self.mask ? (RNSVGMask *)[self.svgView getDefinedMask:self.mask] : nil;
+
+  if (maskNode || filterNode) {
```

Comment:
What was the flow for mask? Was it not working too before? If so, check if there are no issues regarding it that we could close with this PR.